ql/src/test/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/TestHiveMetaStoreAuthorizer.java

```diff
@@ -133,6 +133,22 @@ public void testB_CreateTable_unAuthorizedUser() throws Exception {
     }
   }
 
+  @Test
+  public void testI_CreateExternalTable_authorizedUser() throws Exception {
+    UserGroupInformation.setLoginUser(UserGroupInformation.createRemoteUser(authorizedUser));
+    try {
+      Table table = new TableBuilder()
+              .setTableName(tblName)
+              .setType(TableType.EXTERNAL_TABLE.name())
+              .addCol("name", ColumnType.STRING_TYPE_NAME)
+              .setOwner(authorizedUser)
+              .build(conf);
+      hmsHandler.create_table(table);
```

Comment:
Is this test required? because we are not specifying the external table location here with reduced permissions?

Reply:
Included this as we do not have any create external table test with authorization enabled. We can remove it as we have now added qtest creating an external table with the location specified.